docs/developer/other/API.md

```diff
@@ -106,3 +106,47 @@ Example response:
   }
 }
 ```
+### Portlet Preferences
+
+`/api/prefs/{task}`
```

Comment:
I suggest...

  `/api/v5-6/preferences/{fname}` **AND** `/api/v5-6/preferences/{fname}/windowId`

The first would be for preferences common to the entire publication record, the second for a specific instance of a publication record.

Reply:
@drewwills Ok, so why the windowId? most webcomponents won't have access to their own windowID since they aren't bundled into uportal and I don't know of an easy way for them to access it. Or are you saying to just put the word windowId to differentiate the two?

Reply:
@drewwills Also, are you saying that the first would be the definition ones while the second is the entity? I'm not familiar with what you mean by a publication record.